src/srr-mapping.ts

```diff
@@ -122,6 +125,16 @@ export function handleSRRProvenance(event: SRRProvenanceEvent): void {
   provenance.save()
 }
 
+export function handleSRRProvenanceWithCustomHIstoryId(event: SRRProvenanceWithCustomHIstoryIdEvent): void {
+// Not sure if this will work or not.
+}
+
+export function handleCustomHistory(event: CustomHistoryEvent): void {
+
+  
+}
+
+
```

Comment:
@chatch can you please complete these two function while I'm improving test cases into the startrail code?